src/permissions/index.js

```diff
@@ -45,11 +47,19 @@ const permissions = shield(
     fallbackRule: allow,
 
     // Only allow the external errors when running in the local DEV environment
-    allowExternalErrors: process.env.IS_OFFLINE,
+    // OR when there is a CMR_ERROR a user needs to read
+    allowExternalErrors: allowExternalErrorsBoolean,
```

Comment:
This value does not need to change. It is possible by setting it outside of this shield call you are changing the behavior for future graphql calls. Variables declared outside the scope of lambda functions are cached for the lifetime of the lambda. You wouldn't be able to test that locally.